src/main/java/ro/utcn/sd/boti/stackoverflow/service/StackoverflowService.java

```diff
@@ -0,0 +1,70 @@
+package ro.utcn.sd.boti.stackoverflow.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import ro.utcn.sd.boti.stackoverflow.entity.Question;
+import ro.utcn.sd.boti.stackoverflow.entity.Tag;
+import ro.utcn.sd.boti.stackoverflow.entity.User;
+import ro.utcn.sd.boti.stackoverflow.exception.QuestionNotFoundException;
+import ro.utcn.sd.boti.stackoverflow.exception.TagNotFoundException;
+import ro.utcn.sd.boti.stackoverflow.exception.UserNotFoundException;
+import ro.utcn.sd.boti.stackoverflow.repository.QuestionRepository;
+import ro.utcn.sd.boti.stackoverflow.repository.RepositoryFactory;
+import ro.utcn.sd.boti.stackoverflow.repository.TagRepository;
+
+import java.text.SimpleDateFormat;
+import java.util.*;
+
+@Service
+@RequiredArgsConstructor
+public class StackoverflowService {
```

Comment:
If you will add further features, make sure to create separate services (and not have just one big service for all features).